.github/workflows/pr-name.yml

```diff
@@ -0,0 +1,13 @@
+name: pr-name
+on:
+  pull_request:
+    types: ['opened', 'edited', 'reopened', 'synchronize']
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Dependencies
+      run: npm install @commitlint/config-conventional
+    - uses: JulienKode/pull-request-name-linter-action@v0.2.0
```

Comment:
can we add a step to print the current PR title?

looking through the example failed job, it is hard to tell what the PR title was, and why it failed.

All we have is this:

```
Run JulienKode/pull-request-name-linter-action@v0.2.0
  with:
    configuration-path: ./commitlint.config.js
Error: Error: subject-empty:subject may not be empty
type-empty:type may not be empty
Error: subject-empty:subject may not be empty
type-empty:type may not be empty
```